packages/@webex/plugin-meetings/test/unit/spec/multistream/remoteMediaManager.ts

```diff
@@ -0,0 +1,962 @@
+/* eslint-disable require-jsdoc */
+import EventEmitter from 'events';
+
+import {MediaConnection as MC} from '@webex/internal-media-core';
+import {
+  Configuration,
+  DefaultConfiguration,
```

Comment:
I don't think it's a good idea to use the imported DefaultConfiguration as input for tests, better to copy paste the whole thing, or use make smaller specific configs as input, as it could change, then you wouldn't be testing what you think you are testing.

Reply:
done